src/clusterfuzz/_internal/bot/tasks/commands.py

```diff
@@ -468,7 +470,6 @@ def process_command_impl(task_name, task_argument, job_name, high_end,
     return run_command(task_name, task_argument, job_name, uworker_env)
   finally:
     # Final clean up.
-    if not environment.is_tworker():
-      cleanup_task_state()
+    cleanup_task_state()
```

Comment:
I assume this got removed from #83, since you make sure that this code is not reached by the uworker?

Reply:
Sorry do you mean a different PR?

Reply:
Line of code, sorry

Reply:
I find the question kind of confusing, but I'm no longer relying on that old if statement, the new one is better because the function is called from more than one place.
This doesn't really have anything to do with the untrusted arch.